is-valid/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use failure::Error;
```

Comment:
```suggestion
use failure::*;
```

You want more than just `Error` here.